php/docker/templates/nginx-vhost-conf.d/page-cache.d/10-index.conf

```diff
@@ -44,16 +52,9 @@ set_by_lua_block $skip_cache {
 
     request.query_string = (ngx.var.is_args or '') .. (ngx.var.args or '')
 
-    request.is_woocommerce = (
```

Comment:
Why removing this?

Reply:
It was mentioned in the commit message... 
WooCommerce sets the Cache-Control response header to `no-store no-cache` and some other values, but those two are identified and honored by srcache by default. So it should work out of the box. Also some of those rules were outdated already.